### DISCUSSION TITLE
Archiving a forum?

### DISCUSSION THREAD

**Discussion by GaryM0101** [fudforum/FUDforum#20] [Ideas]
We been discussing shutting down our forum at some point, decline in usage brings up the discussion of archiving the forum and making a read only forum or extracting content and putting it in a form where we wouldn't need the forum software, standard website. I thought possibly doing a script and PDFing the forum in a view of some sort. I know you can currently PDF a topic, but the images don't PDF with it.

Would it be possible to come up with a conversion script to archive or extract a forum into another format?

Cheers,
Gary

**Comment by naudefj** [comment]
Hi Gary,

I've updated the PDF generation script to embed GIF, PNG and JPEG images into the output.
Details @ https://github.com/fudforum/FUDforum/commit/5d68b0ffb159baa8aabe261ceea2073b029c4c4b

Hope it helps.
Frank
